test/powershell/engine/Basic/DefaultCommands.Tests.ps1

```diff
@@ -107,7 +107,6 @@ Describe "Verify approved aliases list" -Tags "CI" {
 "Alias",        "iwmi",                             "Invoke-WMIMethod",                 $($FullCLR                               ),     "ReadOnly",             "",                     ""
 "Alias",        "iwr",                              "Invoke-WebRequest",                $($FullCLR -or $CoreWindows -or $CoreUnix),     "ReadOnly",             "",                     ""
 "Alias",        "kill",                             "Stop-Process",                     $($FullCLR -or $CoreWindows              ),     "",                     "",                     ""
-"Alias",        "lp",                               "Out-Printer",                      $($FullCLR                               ),     "ReadOnly",             "",                     ""
```

Comment:
I'd keep this for FullCLR - this allows us to see version differences.

Reply:
Makes sense, will add back original